src/templates/app/user/security/account_closure.jsx

```diff
@@ -1,37 +1,18 @@
-import React               from 'react';
-// import { Client }          from '../../../../javascript/app/base/client';
-// import { CommonFunctions } from '../../../../javascript/_common/common_functions';
-import {
-    FormRow,
-    SubmitButton }         from '../../../_common/components/forms.jsx';
-import Loading             from '../../../_common/components/loading.jsx';
+import React from 'react';
+import { SubmitButton } from '../../../_common/components/forms.jsx';
+import Loading from '../../../_common/components/loading.jsx';
 
-import { SeparatorLine }   from '../../../_common/components/separator_line.jsx';
-
-const ClosureDescription = ({
-    id,
-    list_items,
-    list_title,
-    subtitle,
-    title,
-    is_trading_limit,
-}) => (
-    <React.Fragment>
-        <h3 className={`${id} ${id && 'invisible'}`}>{title}</h3>
-        <div className={`${id} gr-padding-10 ${id && 'invisible'}`}>
-            <p>{subtitle}</p>
-            {is_trading_limit && <p>{it.L('[_1]Note[_2]: These limits are only applicable to your [_3] real account. To set limits, switch to the respective account and set your limits accordingly.', '<strong>', '</strong>', it.website_name)}</p>}
-            {list_title && <p>{list_title}</p>}
-            {list_items &&
-                <ul className='bullet'>
-                    { list_items.map((item, idx) => (
-                        <li id={item.id} key={idx}>{item.text}</li>
-                    ))}
-                </ul>
-            }
-        </div>
-    </React.Fragment>
-);
+const leaving_reason_list = [
+    { title: 'I have other financial priorities.', id: 'other_priorities' },
+    { title: 'I want to stop myself from trading.', id: 'stop_trading' },
+    { title: 'I’m no longer interested in trading.', id: 'not_interested' },
+    { title: 'I prefer another trading website.', id: 'other_website' },
+    { title: 'The platforms aren’t user-friendly.', id: 'not_user_friendly' },
+    { title: 'Making deposits and withdrawals is difficult.', id: 'difficult' },
+    { title: 'The platforms lack key features or functionality.', id: 'platform_lack' },
+    { title: 'Customer service was unsatisfactory.', id: 'customer_service' },
+    { title: 'I’m deactivating my account for other reasons.', id: 'other' },
```

Comment:
translations are missing

Reply:
The translation was in `return` function but I moved them up.

Reply:
thanks, our script doesn't work with variables and has to be explicitly around the text that you want to translate